src/components/home/welcome/LeftPanel.tsx

```diff
@@ -0,0 +1,50 @@
+import smiliesIcon from 'assets/images/smilies.png'
+import ovalIcon from 'assets/images/oval.png'
+
+import 'components/home/welcome/LeftPanel.css'
+
+const leftPanel = () => {
```

Comment:
Following your codebase and naming convention, this should be like this:
```suggestion
const LeftPanel = () => {
```
WDYT?